tools/termux-installation.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+nav_order: 2
+parent: Applications and Variants
+---
+
+# What is this?
+
+Using this, you can install Termux, a linux "emulator" for Android. The Quest 2 happens to run android, so you can host the SlimeVR server on it using this.
+
+***joycons still require a PC for now and the Wrangler: https://github.com/carl-anders/slimevr-wrangler
+
+# INSTRUCTIONS - QUEST 2
+1. Install a file explorer on your quest, CX File Explorer is good. You can get the APK from https://apkpure.com
+
+2. Download Termux (quest 1/2 is arm64) from here: https://github.com/termux/termux-app/releases
+    DO NOT INSTALL FROM GOOGLE PLAY.
```

Comment:
I don't think you even can do this on a Quest? Either delete this or make this consistent with the other line :P
```suggestion
```
```suggestion
    **DO NOT INSTALL FROM GOOGLE PLAY.**
```